pom.xml

```diff
@@ -562,7 +562,6 @@
             <includeOnlyProperty>^git\.commit\..*$</includeOnlyProperty>
             <includeOnlyProperty>^git\.dirty$</includeOnlyProperty>
             <includeOnlyProperty>^git\.tags$</includeOnlyProperty>
-            <includeOnlyProperty>^git\.total\.commit\.count$</includeOnlyProperty>
```

Comment:
Thank you for this. I'd have thought that the count of the commits leading to up to some tagged commit _is_ reproducible? Or are we catering for the case where a build system makes a shallow clone for efficiency and that truncates the commit history?

Reply:
in fact, if you do a shallow git clone, you get 1 instead of the value of the full content: yes, not so easy to guess :)